.eslintrc.cjs

```diff
@@ -18,6 +18,7 @@ module.exports = {
     "@typescript-eslint/no-explicit-any": "off",
     "@typescript-eslint/no-unused-vars": ["warn", { ignoreRestSiblings: true }],
     "@typescript-eslint/prefer-nullish-coalescing": "warn",
+    "@typescript-eslint/no-duplicate-enum-values": "off",
```

Comment:
I appeared to be getting a false positive here where this warning would throw during pre-commit hooks for the new enums, even though they don't seem to have duplicate values. Adding this to each file also did not work so I was a bit confused why it was getting thrown

Reply:
So disabling the rule here worked whereas disabling it in the file did not